README.md

```diff
@@ -9,14 +11,33 @@ globally. Then run:
 	bower install
 	gulp
 
+
 Develop
 -------
 
 Use `gulp watch` to compile the dist directory in real time.
 
 Remember to save all new dependencies with `--dev`.
 
+
 Deploy
 ------
 
 Statically serve the `dist/` directory.
+
+
+
+## Live
+
+For testing purposes `live.js` has a `testing` variable that mocks events in the near future so that there's no need to keep changing `events.json`. It also reduces the polling rate to 1 second.
+
+New features:
+
+- Desktop notifications
+- Live reload
+- Options panel to filter by type of notifications
+  - 4 types of notifications (provisional): essential, food, events, talks
+  - essential notifications are always sent
+  - food, events and talks can be filtered out depending on your preferences.
+- Options panel also includes a checkbox to switch off background animation.
+- Progress bars!
```

Comment:
I was talking about a progress bar of the full event (or a countdown clock of how many hours missing until the end of Hacking). Looks good though.